content/influxdb/clustered/install/configure-cluster.md

```diff
@@ -821,4 +821,36 @@ spec:
 
 {{% /code-placeholders %}}
 
+### Provide a custom Certificate Authority bundle (optional)
+
+If you do not use a private certificate authority, then you can skip this section.
+
+InfluxDB will attempt to make TLS connections to the services it depends on; notably the Postgresql catalog, and the object store. InfluxDB validates the certificates for all of the connections it makes. If you host these services yourself, and you use a private or otherwise not well-known certificate authority to issue certificates to those services, then InfluxDB will not recognize the issuer and will not be able to validate them. To allow InfluxDB to validate these certificates, you must provide it with a pem bundle containing your custom certificate chain.
+
+First, you must create a config map containing your pem bundle. In the example below, `private_ca.pem` is a pem formatted certificate bundle file. It should be provided to you by your certificate authority administrator. Note that this is *not* the certificate that InfluxDB uses to host its own TLS endpoints. This bundle establishes a chain of trust for the external services that InfluxDB depends on.
+
+```shell
+kubectl -n influxdb create configmap custom-certs --from-file=certs.pem=/path/to/private_ca.pem
+```
+
+{{% note %}}
+
+It's possible to append multiple certificates into the same bundle. This can be helpful if you need to include intermediate certificates, or to explicitly include your leaf certificates. If you do so, leaf certificates should be included before any intermediates they depend on, and the root certificate should be the last one in the bundle.
+
+{{% /note %}}
+
+Then, in your `myinflux.yml`, update the AppInstance `.spec.package.spec.egress` property to refer to that config map. For example:
+
+```yml
+spec:
+  package:
+    spec:
+      egress:
+        customCertificates:
+          valueFrom:
+            configMapKeyRef:
+              key: ca.pem
+              name: custom-ca
+```
```

Comment:
~~~suggestion
1.  Use `kubectl` to create a config map containing your PEM bundle.
    Your certificate authority administrator should provide you with a
    PEM-formatted certificate bundle file.
    
    {{% note %}}
This PEM-formatted bundle file is *not* the certificate that InfluxDB uses to
host its own TLS endpoints. This bundle establishes a chain of trust for the
external services that InfluxDB depends on.
    {{% /note %}}

    In the example below, `private_ca.pem` is the certificate bundle file.

    ```sh
    kubectl --namespace influxdb create configmap custom-ca --from-file=certs.pem=/path/to/private_ca.pem
    ```

    {{% note %}}
It's possible to append multiple certificates into the same bundle.
This can help if you need to include intermediate certificates or explicitly
include leaf certificates. Leaf certificates should be included before any
intermediate certificates they depend on. The root certificate should
be last in the bundle.
    {{% /note %}}

2.  Update your `AppInstance` resource in your `myinfluxdb.yml` to refer to your
    certificate authority config map. Update the `.spec.package.spec.egress`
    property to refer to that config map. For example:

    ```yml
    spec:
      package:
        spec:
          egress:
            customCertificates:
              valueFrom:
                configMapKeyRef:
                  key: ca.pem
                  name: custom-ca
    ```
~~~